packages/components/chip/src/index.ts

```diff
@@ -3,12 +3,19 @@ import { FC } from 'react'
 import { Chip as Root, ChipProps } from './Chip'
 import { ChipClearButton } from './ChipClearButton'
 import { ChipContent } from './ChipContent'
+import { ChipLeadingIcon } from './ChipLeadingIcon'
 
 export { type ChipClearButtonProps } from './ChipClearButton'
 export { type ChipContentProps } from './ChipContent'
+export { type ChipLeadingIconProps } from './ChipLeadingIcon'
 
 ChipClearButton.displayName = 'Chip.ClearButton'
```

Comment:
There is no test env deployed so I can't test, but I think without displayNames for LeadingIcon and other subComponents, markup will be wrong in the "show source" of storybook.

Reply:
for sure stories is on stage 5

Reply:
test stage 6